### PR TITLE
Wide Refactors

### PR DESCRIPTION
**Pull request by jfangonilo**
## What functionality does this accomplish?
N/A - refactors to use prefix syntax, clean up UsersController and SessionsController

### Test Coverage Percentage: 100%
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
- [ ] New Tests added for new features

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Please include a gif of how you feel about this branch:
shrug

### CODE REVIEW

File: spec/features/admin/order_dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# require 'rails_helper'
```
Comment (jfangonilo) [inline spec/features/admin/order_dashboard_spec.rb]: Wrote this test on this branch but ended up mostly refactoring. Will use for the actual branch for 33

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -39,4 +30,23 @@ def destroy
     flash[:success] = "You are now logged out."
     redirect_to "/"
   end
+
```
Comment (jfangonilo) [inline app/controllers/sessions_controller.rb]: Moved all of this from the create action

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -45,41 +47,14 @@ def user_params
...
-    if bool == true
-      @user = @user.update(e_params)
-      flash[:success] = "Your profile has been updated."
-      redirect_to '/profile'
-    else
-      flash[:error] = "This email is already used."
-      render :edit
-    end
-  end
-
```
Comment (jfangonilo) [inline app/controllers/users_controller.rb]: Cleaned a lot of this up in light of Meg's feedback in the last review. All tests still passing

File: app/controllers/cart_controller.rb [hunk 4]
```diff
@@ -14,12 +14,12 @@ def show
 
   def empty
     session.delete(:cart)
-    redirect_to '/cart'
+    redirect_to cart_path
```
Comment (wrenisit) [inline app/controllers/cart_controller.rb]: Great job switching from paths to using resources

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (wrenisit) [reply app/controllers/users_controller.rb]: Wow! That's a lot of cleanup! Well done!

**Pull request merged**